src/Components/Aspire.MongoDB.Driver/ConfigurationSchema.json

```
@@ -0,0 +1,53 @@
+{
+  "definitions": {
+    "logLevel": {
+      "properties": {
+        "MongoDB.Command": {
```

```suggestion
        "MongoDB": {
          "$ref": "#/definitions/logLevelThreshold"
        },
        "MongoDB.Command": {
```

Having this lets someone turn on/off all of `MongoDB` logs at once.